openwisp_monitoring/device/tasks.py

```diff
@@ -63,3 +63,9 @@ def write_device_metrics(pk, data, time=None, current=False):
     except DeviceData.DoesNotExist:
         return
     device_data.writer.write(data, time, current)
+
+
+@shared_task(base=OpenwispCeleryTask)
+def organization_disabled_handler(organization_id):
```

Comment:
similar here, why the `handler` suffix if we don't have anything like that in any of these tasks above?
Let's be consistent with the naming styles used.